src/backend/utils/load/age_load.c

```diff
@@ -63,57 +63,78 @@ static agtype* create_empty_agtype(void) {
 
     memset(&result, 0, sizeof(agtype_in_state));
 
-    result.res = push_agtype_value(&result.parse_state, WAGT_BEGIN_OBJECT, NULL);
-    result.res = push_agtype_value(&result.parse_state, WAGT_END_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_BEGIN_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_END_OBJECT, NULL);
 
     return agtype_value_to_agtype(result.res);
 }
 
-agtype* create_agtype_from_list(char **header, char **fields, size_t fields_len, int64 vertex_id) {
+agtype* create_agtype_from_list(char **header, char **fields,
+                                size_t fields_len, int64 vertex_id) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{